### DISCUSSION TITLE
Switching between relative and absolute line numbers depending on mode

### DISCUSSION THREAD

**Discussion by patleeman** [vscode-neovim/vscode-neovim#1229] [Q&A] [answered]
I use[ this simple neovim plugin](https://github.com/sitiom/nvim-numbertoggle) to toggle line numbers depending on what mode I'm in. 

Normal = Relative
Insert = Absolute

I know VSCode manages the line numbers, but I'm not sure if there is an associated vscode command I can call in the autocommands to set a vscode setting. The setting in question is `"editor.lineNumbers": "relative",`. Has anyone tried toggling settings?

**Comment by theol0403** [comment, marked as answer]
https://github.com/vscode-neovim/vscode-neovim/pull/656
